cores/src/markdown.rs

```diff
@@ -35,7 +42,17 @@ pub async fn store_in_markdown_table(
     let category = generate_categories(&content_md).await?;
     let category_result =
         store_in_category_table(pool, markdown_result.markdown_id, &category).await?;
-    println!("Category result: {category_result:?}");
+
+    let category_markdown_result = sqlx::query_as!(
```

Comment:
Added the query to insert the `markdown_id` && `category_id` to create many to many relationships!